python/cudf/cudf/__init__.py

```diff
@@ -1,9 +1,15 @@
 # Copyright (c) 2018-2019, NVIDIA CORPORATION.
+""" __init__.py
 
-import cupy
+   isort:skip_file
+"""
 
-import rmm
+from cudf.utils.gpu_utils import validate_setup
```

Comment:
Instead of skipping the entire file just skip the import with a `#  isort:skip`

https://github.com/timothycrosley/isort#skip-processing-of-imports-outside-of-configuration